libs/backend/src/intermediate-scripts/reboot-to-vendor-menu

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ -z "${VX_CONFIG_ROOT}" ]]; then
+  echo "VX_CONFIG_ROOT is not set likely because running in dev" >&2
+else
+  echo 1 > "${VX_CONFIG_ROOT}/REBOOT_TO_VENDOR_MENU"
```

Comment:
I'm going to actually read from this file and set up all the relevant user permissions in a follow-up vxsuite-complete-system PR

Reply:
https://github.com/votingworks/vxsuite-complete-system/pull/395